portal/static/js/src/empro.js

```diff
@@ -0,0 +1,63 @@
+import EMPRO_DOMAIN_MAPPINGS from "./data/common/empro_domain_mappings.json";
+import tnthAjax from "./modules/TnthAjax.js";
+
+var emproObj = function() {
+    this.domains = [];
+    this.hasHardTrigger = false;
+    this.hasSoftTrigger = false;
+    this.userId = 0;
+};
+emproObj.prototype.populateDomainDisplay = function() {
+    this.domains.forEach(domain => {
```

Comment:
given the fact that multiple domains map to a single display - do you need to confirm you haven't already added, so we don't say get `mood_changes` twice?

Reply:
yeah, I removed duplication(s) here:  https://github.com/uwcirg/truenth-portal/blob/cd5dff64e6d014719a7299e6a446dce06962c16c/portal/static/js/src/empro.js#L32